installation/sql/postgresql/base.sql

```diff
@@ -1149,7 +1155,8 @@ CREATE INDEX "#__workflow_stages_idx_checked_out" ON "#__workflow_stages" ("chec
 --
 
 INSERT INTO "#__workflow_stages" ("id", "asset_id", "ordering", "workflow_id", "published", "title", "description", "default") VALUES
-(1, 57, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1);
+(1, 57, 1, 1, 1, 'COM_WORKFLOW_BASIC_STAGE', '', 1),
+(2, 90, 1, 2, 1, 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES', '', 1);
```

Comment:
Is the title 'COM_WORKFLOW_BASIC_WORKFLOW_MODULES' of the workflow the right description for the stage here?

Reply:
Thanks, pushed some new commits to fix this